hw/pci-bridge/cxl_socket_transport.c

```diff
@@ -558,16 +582,16 @@ void wait_for_cxl_io_cfg_completion(int socket_fd, uint16_t tag, uint32_t *data)
                 assert(entry->packet_size ==
                            sizeof(cxl_io_completion_packet_t) ||
                        entry->packet_size ==
-                           sizeof(cxl_io_completion_data_packet_t));
+                           sizeof(cxl_io_completion_data_packet_32b_t));
             }
```

Comment:
It would never be `cxl_io_completion_data_packet_64b_t`?

Reply:
Yes. In fact, the function and the call stack ask for a 32-byte integer.